cli/genpolicy/genpolicy.go

```diff
@@ -81,6 +82,22 @@ func (r *Runner) Run(ctx context.Context, yamlPath string, cmPath []string, logg
 		return fmt.Errorf("running genpolicy: %w", err)
 	}
 
+	data, err := os.ReadFile(yamlPath)
+	if err != nil {
+		return fmt.Errorf("read %s: %w", yamlPath, err)
+	}
+	kubeObjs, err := kuberesource.UnmarshalApplyConfigurations(data)
+	if err != nil {
+		return fmt.Errorf("unmarshal %s: %w", yamlPath, err)
+	}
+	resource, err := kuberesource.EncodeResources(kubeObjs...)
+	if err != nil {
+		return err
+	}
+	if err := os.WriteFile(yamlPath, resource, 0o644); err != nil {
+		return fmt.Errorf("write output file: %w", err)
+	}
+
```

Comment:
Why do we need the formatting both in genpolicy as well as in generate?

Reply:
Both write the files out. My suggestion would be to write a wrapper function around this and just call it from both.

Reply:
Edit: oh, you mean why do we need to read, encode, and write again? As far as I can tell, the formatting info is lost in the bytes representation read from the file, quotes and newlines get escaped.